packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -0,0 +1,292 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'button.dart';
+
+// Padding around the line at the edge of the text selection that has 0 width and
+// the height of the text font.
+const double _selectionHandlesPadding = 18.0;
+const double _kToolbarScreenPadding = 8.0; // pixels
```

Comment:
nit: I found this comment a bit confusing since all padding is pixels and was wondering if it is trying to specify logical vs physical? I realized it's probably copy-pasted from previous code, but the comment isn't on the line above. May be better to remove it.